docs/running-on-yarn.md

```diff
@@ -60,6 +60,14 @@ Running Spark on YARN requires a binary distribution of Spark which is built wit
 Binary distributions can be downloaded from the [downloads page](http://spark.apache.org/downloads.html) of the project website.
 To build Spark yourself, refer to [Building Spark](building-spark.html).
 
+To make Spark runtime jars accessible from YARN side, basically you will have 3 options:
```

Comment:
Thanks a lot @vanzin for your suggestion, actually the reason I want to emphasize here is that we want deliver a better approach for users to start spark on yarn. I will reorganize the words. Thanks a lot.